### PR TITLE
Fix docker release workflow

### PR DESCRIPTION
**Pull request by coderobe**
This should fix the broken docker workflow.

I've tested these changes locally

### CODE REVIEW

**Review by Nexus2k — APPROVED** [review]

**Review by EgorPopelyaev — APPROVED** [review]

File: scripts/dockerfiles/polkadot_injected_release.Dockerfile [hunk 1]
```diff
@@ -25,15 +25,15 @@ RUN apt-get update && \
 		libssl1.1 \
 		ca-certificates \
 		gnupg && \
-	useradd -m -u 1000 -U -s /bin/sh -d /polkadot polkadot && \
+	useradd -m -u 1000 -U -s /bin/sh -d /polkadot polkadot
 # add repo's gpg keys and install the published polkadot binary
-	gpg --recv-keys --keyserver ${GPG_KEYSERVER} ${POLKADOT_GPGKEY} && \
+RUN gpg --keyserver ${GPG_KEYSERVER} --recv-keys ${POLKADOT_GPGKEY} && \
```
Comment (alvicsam) [inline scripts/dockerfiles/polkadot_injected_release.Dockerfile:30]: Is it needed to divide one `RUN` into several? I thought all commands were in one `RUN` in order to have less layers

Comment (coderobe) [reply scripts/dockerfiles/polkadot_injected_release.Dockerfile:30]: No, i suppose i can join them again. I think i did that for debugging purposes.

**Review by alvicsam — APPROVED** [review]

**Comment by coderobe** [comment]
bot merge

**Comment by coderobe** [comment]
unrelated failure

**Pull request merged**